### PR TITLE
New changes been made in the train file

### PR DESCRIPTION
**Pull request by harisalam**
Now the code cleans the data perfectly.

### CODE REVIEW

File: src/train/base_model.py [hunk 1]
```diff
@@ -16,8 +14,8 @@ class ITrain(metaclass=ABCMeta):
     # and these kinds of punctuations
     # BTW： maybe use regular expression
     # TODO: keep maintaining for this object
-    FILTER_WORD = ['<root>', ',', '，', '.', '。', ';', '；', '!', '！', '?', '？',
-                   '\\', '、', '“', '”', '……']
+    # FILTER_WORD = ['<root>', ',', '，', '.', '。', ';', '；', '!', '！', '?', '？',
```
Comment (hvkone) [inline src/train/base_model.py:17]: Did you add different punctuation?

Comment (harisalam) [reply src/train/base_model.py:17]: NO!
I've replaced it with "string.punctuation" library and added some extra data cleaning functionality. Previously the data was not been cleaned by the function itself.

File: src/train/train_model.py [hunk 2]
```diff
@@ -31,31 +31,25 @@ def __init__(self, language_name, pre_model_name, our_corpus_name):
...
         """
         data is one or several sentence(s) we expect
-
         if data is \n, \t, empty str, etc, replace them
-
         :param data: raw data
         :return: data after cleaning
         """
-        cleaned_data = re.sub('[\n\t]+', '', data)
+        cleaned_data = re.sub('\w*\d\w*', '', data)
```
Comment (jackzhenguo) [inline src/train/train_model.py:49]: Yes, this is a good optimize for clean task

**Comment by jackzhenguo** [comment]
@harisalam right now, your commit has some conflicts with our master branch of this repo. So I suggest you do something following in git window command:

plz do it at your master branch of forked repo as  following operations

1. git remote add upstream https://github.com/hvkone/wordfinder.git
2. git checkout master
3. git merge upstream master
4. git checkout -b newfeature
5. git add .
6. git commit -m "my new feature"
7. git push origin newfeature

final pull request

**Comment by jackzhenguo** [comment]
@hvkone Willie, this pull request has some conflicts with our base master branch, so merge it until conflicts fix.

**Review by hvkone — APPROVED** [review]
The edits look good. Excited to implement them.

**Comment by harisalam** [comment]
Willie can you please revert the merged changes, you've been told by Zhen that my pull request has some conflicts don't merge it

**Comment by harisalam** [comment]
I'm working on it and will initiate another pull request in a while, but for the proper working of the project, you've to revert the recently closed pull request.

**Pull request merged**